src/applications/disability-benefits/2346/Components/VeteranInformationPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getVeteranInformationData } from '../actions';
+
+class VeteranInformationPage extends React.Component {
+  componentDidMount() {
+    this.props.getVeteranInformationData();
+  }
+  render() {
+    const { first, last, gender, dateOfBirth } = this.props;
+    return (
+      <div>
+        <p>This is the personal information we have for you.</p>
+        <div>
+          <div className="usa-alert schemaform-sip-alert">
+            <div className="usa-alert-body">
+              <p className="vads-u-margin--1px">
+                {first} {last}
+              </p>
+              <p className="vads-u-margin--1px">Date of Birth: {dateOfBirth}</p>
```

Comment:
`dataOfBirth` might be in the format `YYYY-MM-DD`, you'll need to include moment to reformat this to `mm/dd/yyyy`

Reply:
Done